<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="dataTreeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Logs</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Overview</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Remote location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Last backup:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startJobButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Start backup now!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Open in Finder</string>
      </property>
     </widget>
     <widget class="QLabel" name="locationLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="remoteLocationLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lastBackupLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Details</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>351</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Job Information</string>
      </property>
      <widget class="QLabel" name="jobStatusLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>finished</string>
       </property>
      </widget>
      <widget class="QLabel" name="dateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>341</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="dataSizeLabel">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Size: 0kb</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="startStopButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentJob">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionAdd_new_job"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_new_job">
   <property name="text">
    <string>Add new job</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
